src/functions/module_info.ts

```diff
@@ -1,7 +1,58 @@
+import { getReference as mergeReference, splitReference } from "../helpers";
+import { setXcToFixedReferenceType } from "../helpers/reference_type";
 import { _t } from "../translation";
-import { AddFunctionDescription, CellValue, Maybe } from "../types";
+import { AddFunctionDescription, Maybe } from "../types";
 import { CellErrorType, NotAvailableError } from "../types/errors";
+import { CellValue } from "./../types/cells";
 import { arg } from "./arguments";
+import { assert, toString } from "./helpers";
+
+// -----------------------------------------------------------------------------
+// CELL
+// -----------------------------------------------------------------------------
+export const CELL = {
+  description: _t("Gets information about a cell."),
+  args: [
+    arg(
+      "info_type (string)",
+      _t('The type of information requested. Can be "address", "col", "contents" or "row".')
+    ),
+    arg("reference (meta)", _t("The reference to the cell.")),
+  ],
+  returns: ["ANY"],
+  compute: function (info: CellValue, reference: string) {
+    // NOTE: missing from Excel: "color", "filename", "format", "parentheses", "prefix", "protect", "type" and "width"
```

Comment:
'format' and 'type' shouldn't be complex to implement.

'format' could be useful for us.